samwise.js

```diff
@@ -1,11 +1,10 @@
 /**
```

Comment:
SCRIPT-I'm impressed with your use of design patterns here, it really enhances the code organization.